buildSrc/src/main/kotlin/otel.jmh-conventions.gradle.kts

```diff
@@ -0,0 +1,36 @@
+plugins {
+  id("me.champeau.jmh")
+  id("io.morethan.jmhreport")
+}
+
+dependencies {
+  jmh(platform(project(":dependencyManagement")))
+  jmh("org.openjdk.jmh:jmh-core")
+  jmh("org.openjdk.jmh:jmh-generator-bytecode")
+}
+
+// invoke jmh on a single benchmark class like so:
+//   ./gradlew -PjmhIncludeSingleClass=StatsTraceContextBenchmark clean :grpc-core:jmh
+jmh {
+  failOnError.set(true)
+  resultFormat.set("JSON")
+  // Otherwise an error will happen:
+  // Could not expand ZIP 'byte-buddy-agent-1.9.7.jar'.
+  includeTests.set(false)
+  profilers.add("gc")
+  val jmhIncludeSingleClass: String? by project
+  if (jmhIncludeSingleClass != null) {
+    includes.add(jmhIncludeSingleClass as String)
```

Comment:
Just curious: why this `as String` is needed? It is already clear that this is not null.

Reply:
IIRC, the `by` syntax seems to result in something "dynamic". Seems like it might be a Gradle bug, but it makes kotlin not treat the null check as a type cast.